From 2e0eaea69065dd93f3733b2eeb4a95d896f6ba96 Mon Sep 17 00:00:00 2001
From: Shawn W Dunn <sfalken@opensuse.org>
Date: Wed, 24 Dec 2025 07:44:53 -0800
Subject: [PATCH 3/7] configuration changes for default niri session

Set openSUSE default background
Disable "natural scrolling" for scrollwheel in default session
---
 configurations/lxqt-niri.kdl  | 2 +-
 configurations/niri/input.kdl | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/configurations/lxqt-niri.kdl b/configurations/lxqt-niri.kdl
index b222b52..5f6b12f 100644
--- a/configurations/lxqt-niri.kdl
+++ b/configurations/lxqt-niri.kdl
@@ -14,7 +14,7 @@
 // See the binds section below for more spawn examples.
 // spawn-at-startup "waybar"
 // To run a shell command (with variables, pipes, etc.), use spawn-sh-at-startup:
-   spawn-sh-at-startup "swaybg -i /usr/share/lxqt/wallpapers/waves-logo.png"
+   spawn-sh-at-startup "swaybg -i /usr/share/wallpapers/openSUSEdefault/contents/images/default.png"
 // The above sets the wallpaper used in the overview.
    spawn-sh-at-startup "lxqt-session && niri msg action quit -s"
 
diff --git a/configurations/niri/input.kdl b/configurations/niri/input.kdl
index 3f865cb..9e8eabc 100644
--- a/configurations/niri/input.kdl
+++ b/configurations/niri/input.kdl
@@ -30,7 +30,7 @@ input {
         dwtp
         // drag false
         // drag-lock
-        natural-scroll
+        // natural-scroll
         accel-speed 0.6
         // accel-profile "flat"
         // scroll-method "two-finger"
-- 
2.52.0

